newsfragments/2629.misc.rst

```diff
@@ -0,0 +1 @@
+Use constant for loopback address across the codebase.
```

Comment:
Have a look at the towncrier documentation (https://github.com/twisted/towncrier#philosophy). IMO this PR does not need a newsfragment.  If we had a formal changelog I think it might be better suited there.  In any case, I prefer to err on the side of verbosity.